supplier/tests/test_supplier_page.py

```diff
@@ -39,3 +93,77 @@ def test_supplier_check_false(non_supplier_mock):
 def test_supplier_check_true(saved_supplier0):
     assert saved_supplier0 in Supplier.objects.all()
     assert get_supplier(saved_supplier0) == saved_supplier0
+
+
+@pytest.mark.django_db()
+def test_get_product_return_supplier_product(request_searched_product_mock,
+                                             saved_supplier0,
+                                             saved_supplier_product0,
+                                             saved_supplier_product1,
+                                             searched_product_string):
+    assert saved_supplier_product0.user_name == saved_supplier0
+    assert saved_supplier_product1.user_name == saved_supplier0
```

Comment:
Why do we also need `saved_supplier_product1` here? 
I guess you are trying to show that this supplier has another product, but I don't think it is necessary.

Reply:
I think he is checking here that both products has the same supplier before doing the test so if there is a problem that the saved_supplier0 doesnt have one of the products it will assert False

Reply:
I have more notes to add here (to this file), but since we are only a couple of hours from the deadline I think it is more important to merge this PR than to fix these small things.

Reply:
like karam said, i wanted to prove that there are two supplier products, and that the search does bring one of them.